src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -61,13 +61,17 @@
 
 #include <U2Gui/GUIUtils.h>
 
+#include <U2View/ADVSequenceObjectContext.h>
+#include <U2View/AnnotatedDNAView.h>
 #include <U2View/DetView.h>
 #include <U2View/McaEditorReferenceArea.h>
 
+#include "../../circular_view/src/CircularViewSplitter.h"
```

Comment:
Cross-plugin deps are illegal ( a shared lib should be used)
But you do not no need this at all. The way you use splitter does not require any deps beyond qt.

Reply:
done